cli/ioctl/cmd/account/account.go

```diff
@@ -9,24 +9,26 @@ package account
 import (
 	"bytes"
 	"context"
+	"encoding/hex"
 	"fmt"
 	"syscall"
 
 	"github.com/ethereum/go-ethereum/accounts/keystore"
-	"github.com/iotexproject/go-pkgs/crypto"
-	"github.com/iotexproject/iotex-address/address"
-	"github.com/iotexproject/iotex-proto/golang/iotexapi"
-	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
```

Comment:
use go-pkgs/crypto for message signing
we don't want to introduce new dependency here